cmake/Macros.cmake

```diff
@@ -244,3 +257,102 @@ if(CMAKE_VS_PLATFORM_NAME STREQUAL "Tegra-Android")
         set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
     endmacro()
 endif()
+
+# Find the requested package and make an INTERFACE library from it
+# Usage: sfml_find_package(wanted_target_name [REQUIRED]
+#                          [HOST_SEARCH]                    # Use find_host_package() instead of find_package()
```

Comment:
I'd think this would be better in a separate function, mirroring the original, i.e. `sfml_find_host_package()`.

Reply:
Is there a way to detect when we want host package and when we don't? I'd rather just remove that 'host' thing and search appropriatly in a transparent way